Formula/o/openjdk.rb

```diff
@@ -1,8 +1,9 @@
 class Openjdk < Formula
   desc "Development kit for the Java programming language"
   homepage "https://openjdk.org/"
-  url "https://github.com/openjdk/jdk24u/archive/refs/tags/jdk-24.0.2-ga.tar.gz"
-  sha256 "d5a1b364de4335d86590d2e91eb8fce0560ade28759734c531915b8293e502e8"
+  url "https://github.com/openjdk/jdk25u/archive/refs/tags/jdk-25+36.tar.gz"
+  version "25"
+  sha256 "8722bf578a77261ffbb26011c339e91571ec0474f1bcfea40188dd5451aa1d69"
```

Comment:
GA tag is now available
```suggestion
  url "https://github.com/openjdk/jdk25u/archive/refs/tags/jdk-25-ga.tar.gz"
  sha256 "7b84e7c96086b4e75ee128921d301427012447bdfbe1741398725ab6218afe11"
```